ansible_collections/arista/avd/roles/eos_designs/doc/fabric-topology.md

```diff
@@ -215,6 +215,13 @@ defaults <- node_group <- node_group.node <- node
     # Point-to-Point interface speed - will apply to uplinks on both ends | Optional.
     uplink_interface_speed: < interface_speed | forced interface_speed | auto interface_speed >
 
+    # Custom structured config applied to "uplink_interfaces", and "uplink_switch_interfaces"
+    # When uplink_type == "p2p", custom structured config added under ethernet_interfaces.<interface> for eos_cli_config_gen
+    # Overrides the settings on the ethernet interface level.
+    # When uplink_type == "port-channel", custom structured config added under port_channel_interfaces.<interface> for eos_cli_config_gen
+    # Overrides the settings on the port-channel interface level.
```

Comment:
```suggestion
    # Overrides the settings on the port-channel interface level.
    # "uplink_structured_config" is applied after "structured_config", so it can override "structured_config" defined on node-level.
```